patternlab/styleguide/source/assets/js/modules/stickyTOC.js

```diff
@@ -3,7 +3,7 @@ export default (function (window,document,$,undefined) {
   $('.pre-content .ma__sticky-toc').each(function() {
     const $toc = $('.ma__sticky-toc'),
           $tocContent = $('.ma__sticky-toc__links'),
-          $tocSections = $('.pre-content').next().find('.ma__comp-heading'),
+          $tocSections = $('.ma__information-details .page-content, .ma__stacked-row-section__container .ma__stacked-row-section__title').find('h2'),
```

Comment:
@tnathanjames For info details page, if `.ma__rich-text` under `.page-content` contains `h2`, the JS would pick it up with this code.  Here is the sample I created on feature5:  http://massgovfeature5.prod.acquia-sites.com/info-details/how-where-to-recycle

<img width="570" alt="how___where_to_recycle___mass_gov" src="https://user-images.githubusercontent.com/9633303/44348691-28554b80-a469-11e8-9557-8ed6c44ea772.png">

It's not supposed to have h2, but since we cannot control what authors put in RTE, it would be safe to exclude the `.ma__rich-text`.

Curated list wouldn't be the case since `<h2>` is in the separate container `.ma__stacked-row-section__title` from its associated body content `.main-content`.

Reply:
@ygannett Since this is something that is broken in production and since this PR is just reverting a previous change, we don't want to put additional changes into this. We don't want h3's in the content, but it could be seen as a feature instead of a bug to allow content authors to add additional headings to the TOC. I think we should proceed with this and we can come back for exclusion of h2's in rich text if we decide we want to later.

Reply:
ok.